### PR TITLE
Add support for Dropwizard 2.0.x

### PR DESCRIPTION
**Pull request by joschi**
This change set adds support for Dropwizard 2.0.x while retaining support for Dropwizard 1.3.x in the `dropwizard1` sub-project.

Refs #2554

### CODE REVIEW

**Comment by trustin** [comment]
Thanks! That was really quick, @joschi. Did you have a chance to try this suggestion, though? - https://github.com/line/armeria/issues/2554#issuecomment-595217934

**Comment by joschi** [comment]
@trustin Thanks for the pointer! I missed that comment and will update this PR accordingly. 👍

File: dropwizard1/build.gradle [hunk 1]
```diff
@@ -0,0 +1,62 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath "com.github.jengelman.gradle.plugins:shadow:${managedVersions['com.github.jengelman.gradle.plugins:shadow']}"
+    }
+}
```
Comment (trustin) [inline dropwizard1/build.gradle]: Do we need this? Seems unused to me.

**Comment by trustin** [comment]
BTW, please ignore the build failure against Java 11. We have a flaky test.

**Comment by joschi** [comment]
@trustin I've removed the unused buildscript from "dropwizard" and "dropwizard1" and rebased the branch on current `master`.

**Comment by trustin** [comment]
Let me take care of the build failure later.

**Comment by trustin** [comment]
Merged. :clap::clap::clap: to @joschi!

**Comment by joschi** [comment]
@trustin @ikhoon @minwoox Thanks everyone for the smooth experience contributing to Armeria.

Keep up the good work!  👍

**Comment by cricket007** [comment]
@joschi @trustin 

Should this page be updated? 
https://line.github.io/armeria/advanced-dropwizard-integration.html

**Comment by trustin** [comment]
Oh, yes, indeed. :sweat_smile:

**Comment by trustin** [comment]
@OneCricketeer The site has been fully updated: https://line.github.io/armeria/docs/advanced-dropwizard-integration/

**Comment by OneCricketeer** [comment]
looks great!

**Pull request merged**